collect_app/src/test/java/org/odk/collect/android/fragments/dialogs/FormsDownloadResultDialogTest.kt

```diff
@@ -0,0 +1,206 @@
+package org.odk.collect.android.fragments.dialogs
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AlertDialog
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.CoreMatchers.nullValue
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.odk.collect.android.R
+import org.odk.collect.android.application.Collect
+import org.odk.collect.android.formmanagement.downloaderror.FormsDownloadErrorItem
+import org.odk.collect.fragmentstest.DialogFragmentTest
+import org.odk.collect.testshared.RobolectricHelpers
+import org.robolectric.Shadows
+
+@RunWith(AndroidJUnit4::class)
+class FormsDownloadResultDialogTest {
+    private val downloadErrorItem = FormsDownloadErrorItem("", "", "", "")
+    val listener = mock<FormsDownloadResultDialog.FormDownloadResultDialogListener>()
+
+    @Test
+    fun `The dialog should have no title`() {
```

Comment:
I don't think we need this test. If we apply Callum Stott's Lazy Programmer Principle™️ (i.e. programmers won't write code they don't have to) then there isn't going to be a title unless we have a test that enforces there being one.

Reply:
ok, is it a part of a book or online course somewhere? :) I have two weeks off ahead of me so I could read something :)

Reply:
Done.